README.md

```diff
@@ -1,2 +1,14 @@
 # ruuvi.node_nrf91.c
 nRF91 project for Ruuvi Node
+ 
+# Cloning and building
+mkdir ruuvi_node_sdk && cd ruuvi_node_sdk
+west init -m https://github.com/ruuvi/ruuvi.node_nrf91.c
+west update
+cd ruuvi_node
+west build -b nrf9160_pca10090ns
```

Comment:
I have set the board as nrf9160_pca10090ns so "west build" is only required

Reply:
Should we alaso add the instructions to install the toolchain.

Reply:
Fixed these in #a157c5f